src/components/list/list.scss

```diff
@@ -127,7 +127,8 @@ $list-border-radius: 0.375rem; // 6px
 
     &.mdc-deprecated-list--two-line {
         .mdc-deprecated-list-item__text {
-            padding: functions.pxToRem(8) 0;
+            padding-top: functions.pxToRem(8);
+            padding-bottom: functions.pxToRem(8);
```

Comment:
Yeah, when we _do_ override properties, we should probably try to override as few properties as possible, and make sure to only override those properties we really mean to. So using the shorthand css property should basically be banned when overriding MDC styles.